### PR TITLE
Add webots version in database for scenes, animations and simulations

### PR DESCRIPTION
**Pull request by omichel**
As discussed, to facilitate the support for different versions of Webots from R2022b.

### CODE REVIEW

File: httpdocs/ajax/animation/create.php [hunk 1]
```diff
@@ -63,12 +63,14 @@ function parse_mf_string($line, $parameter) {
       $title = parse_sf_string($line, 'title');
       $info = parse_mf_string($line, 'info');
       $description = implode("\n", $info);
-    }
+    } else if (substr($line, 0, 30) == '<meta name="version" content="')
```
Comment (ThomasOliverKimble) [inline httpdocs/ajax/animation/create.php:66]: The x3d files I am testing with have: `<meta name="generator" content="Webots" />`
Will this be a problem? I have not come across this meta header node yet.

Comment (omichel) [reply httpdocs/ajax/animation/create.php:66]: Yes, @BenjaminDeleze is implementing it right now as discussed. So, I am anticipating on the existence of this meta node.

**Review by ThomasOliverKimble — APPROVED** [review]
Looks good to me, waiting on @BenjaminDeleze and his implementation of version in meta header of the x3d files

**Review by BenjaminDeleze — APPROVED** [review]
Thank you

**Comment by ThomasOliverKimble** [comment]
Thank you

**Pull request merged**